swap_meet/vendor.py

```diff
@@ -1,2 +1,73 @@
-class Vendor:
-    pass
+from swap_meet.item import Item
+
+class Vendor():
+    def __init__(self, inventory=[]): # if the vendor has a list use it, if not, use the default empty list
+        # if not inventory: # take optional inventory by checking if the inventory is valid, (has a list of things)
+        #     # self.inventory = [] # create list
+        # else: 
+        self.inventory = inventory
+        self.item = Item()
```

Comment:
Curios why you added` self.item = Item()`.
You do not need it here because we will be passing in an inventory and iterating through that inventory(items) in our methods.